### ISSUE TITLE
Dependency basicspace was removed from CRAN

### ISSUE THREAD

**Issue by kuriwaki** [wmay/dwnominate#25]
The dependency basicspace was removed from CRAN. The archive does not indicate that the package started to fail, though. CRAN just suggests the maintainer was unreachable.  

https://cran.r-project.org/package=basicspace

Therefore, the installation will not work. 

However, I have managed to download it via:

```
remotes::install_version("basicspace", version = "0.24")
```

And then 

`remotes::install_github('wmay/dwnominate')` should work.

**Comment by wmay** [comment]
Yeah, the maintainer of some of voteview's old packages changed jobs and no one seems to know how to contact him. I've been meaning to email the voteview people about this, since it's their package.

In the meantime, you can install basicspace from the archived tar.gz files. There's nothing wrong with it aside from some minor formatting issue.
